src/opensourceReminder.job.ts

```diff
@@ -51,6 +41,32 @@ async function reminderCallback() {
     log.info('Succesfully sent open source reminder messages');
 }
 
+async function getOpenSourceSubscribedUsers(): Promise<Collection<string, GuildMember>> {
+    log.info(`Getting open source subscribed users`);
```

Comment:
these could be normal strings, but not a blocker

Reply:
ohh true, I missed that detail 😅. I'll change it later

Reply:
thank you 👍